env.template

```diff
@@ -14,10 +14,9 @@ ASSISTANT_ROLE="CHANGEME"
 LIBRARIAN_CORPUS=CHANGEME
 LIBRARIAN_COLLECTION=knowledge-base
 
-# Ollama LLM Runtime
-OLLAMA_HOST=CHANGEME
-OLLAMA_PORT=11434
-OLLAMA_CHAT_MODEL=mistral
+# OPENAI LLM Runtime
+OPENAI_API_KEY=sk-90XlXxxxxxxxxxxxx
+OPENAI_CHAT_MODEL=gpt-4
```

Comment:
Please go through `readme.md` too, documenting all the variables we use.

I also wrote quite extensively about deployment and that section is now outdated. Kindly remove the Ollama bits from the documentation and update the deployment bits to be proper.

Reply:
Please review the readme